astroquery/esasky/core.py

```diff
@@ -48,6 +48,7 @@ class ESASkyClass(BaseQuery):
     __IS_SURVEY_MISSION_STRING = "isSurveyMission"
     __ZERO_ARCMIN_STRING = "0 arcmin"
     __MIN_RADIUS_CATALOG_STRING = "5 arcsec"
+    __DEFAULT_SOURCE_LIMIT = "default"
```

Comment:
We could drop the `__` here and make this a `ConfigurationItem` and make it more accessible to users